src/assets/scss/application.scss

```diff
@@ -83,3 +83,8 @@ $govuk-assets-path: "/public/";
   margin-top: 0; 
   margin-bottom: 20px;
 }
+
+.instruction {
```

Comment:
What do you think about using BEM naming to make it really clear when we are adding custom CSS? A name maybe like `bav__instruction`?

Reply:
Done